templates/monitoring/metric_tag_configuration.tf.tpl

```diff
@@ -0,0 +1,86 @@
+    "{{ .Config.Name }}.go_gc_heap_tiny_allocs_objects_total": "count",
+    "{{ .Config.Name }}.go_gc_pauses_seconds_total": "distribution",
+    "{{ .Config.Name }}.go_goroutines": "gauge",
+    "{{ .Config.Name }}.go_info": "gauge",
+    "{{ .Config.Name }}.go_memory_classes_heap_free_bytes": "gauge",
+    "{{ .Config.Name }}.go_memory_classes_heap_objects_bytes": "gauge",
+    "{{ .Config.Name }}.go_memory_classes_heap_released_bytes": "gauge"
+    "{{ .Config.Name }}.go_memory_classes_heap_stacks_bytes": "gauge",
+    "{{ .Config.Name }}.go_memory_classes_heap_unused_bytes": "gauge",
+    "{{ .Config.Name }}.go_memory_classes_metadata_mcache_free_bytes": "gauge",
+    "{{ .Config.Name }}.go_memory_classes_metadata_mcache_inuse_bytes": "gauge",
+    "{{ .Config.Name }}.go_memory_classes_metadata_mspan_free_bytes": "gauge",
+    "{{ .Config.Name }}.go_memory_classes_metadata_mspan_inuse_bytes": "gauge",
+    "{{ .Config.Name }}.go_memory_classes_metadata_other_bytes": "gauge",
+    "{{ .Config.Name }}.go_memory_classes_os_stacks_bytes": "gauge",
+    "{{ .Config.Name }}.go_memory_classes_other_bytes": "gauge",
+    "{{ .Config.Name }}.go_memory_classes_profiling_buckets_bytes": "gauge",
+    "{{ .Config.Name }}.go_memory_classes_total_bytes": "gauge",
+    "{{ .Config.Name }}.go_memstats_alloc_bytes": "gauge",
+    "{{ .Config.Name }}.go_memstats_alloc_bytes_total": "count",
+    "{{ .Config.Name }}.go_memstats_buck_hash_sys_bytes": "gauge",
+    "{{ .Config.Name }}.go_memstats_frees_total": "count",
+    "{{ .Config.Name }}.go_memstats_gc_cpu_fraction": "gauge",
+    "{{ .Config.Name }}.go_memstats_gc_sys_bytes": "gauge",
+    "{{ .Config.Name }}.go_memstats_heap_alloc_bytes": "gauge",
+    "{{ .Config.Name }}.go_memstats_heap_idle_bytes": "gauge",
+    "{{ .Config.Name }}.go_memstats_heap_inuse_bytes": "gauge",
+    "{{ .Config.Name }}.go_memstats_heap_objects": "gauge",
+    "{{ .Config.Name }}.go_memstats_heap_released_bytes": "gauge",
+    "{{ .Config.Name }}.go_memstats_heap_sys_bytes": "gauge",
+    "{{ .Config.Name }}.go_memstats_last_gc_time_seconds": "gauge",
+    "{{ .Config.Name }}.go_memstats_lookups_total": "count",
+    "{{ .Config.Name }}.go_memstats_mallocs_total": "count",
+    "{{ .Config.Name }}.go_memstats_mcache_inuse_bytes": "gauge",
+    "{{ .Config.Name }}.go_memstats_mcache_sys_bytes": "gauge",
+    "{{ .Config.Name }}.go_memstats_mspan_inuse_bytes": "gauge",
+    "{{ .Config.Name }}.go_memstats_mspan_sys_bytes": "gauge",
+    "{{ .Config.Name }}.go_memstats_next_gc_bytes": "gauge",
+    "{{ .Config.Name }}.go_memstats_other_sys_bytes": "gauge",
+    "{{ .Config.Name }}.go_memstats_stack_inuse_bytes": "gauge",
+    "{{ .Config.Name }}.go_memstats_stack_sys_bytes": "gauge",
+    "{{ .Config.Name }}.go_memstats_sys_bytes": "gauge",
+    "{{ .Config.Name }}.go_sched_goroutines_goroutines": "gauge",
+    "{{ .Config.Name }}.go_sched_latencies_seconds": "distribution",
+    "{{ .Config.Name }}.go_threads": "gauge",
```

Comment:
It would be nice to add a block here and in the tags section. That way teams can expand this list as they see fit.

Maybe also a third block at the bottom of the file that would allow completely custom `datadog_metric_tag_configuartion` blocks.

Reply:
Done